tools/privacyidea-token-janitor

```diff
@@ -431,16 +431,44 @@ def find(last_auth, assigned, active, tokeninfo_key, tokeninfo_value,
                         token_obj.token.serial,
                         token_obj.token.tokentype,
                         token_obj.token.description,
-                        token_obj.get_tokeninfo()))
+                        token_obj.get_tokeninfo()
+                        ))
+                    user = token_obj.user
```

Comment:
Thanks a lot for the PR.
With the ``user = token_obj.user`` you are *always*, when the token is assigned to a user, also fetching user from the user directory. This can take a long time! So we should have a parameter to explicitly require the username to be printed.